CoroutineNetworkDataSourceAdapter.md

```diff
@@ -1,7 +1,10 @@
-# Coroutine Network Data Source Adapter
-The `CoroutineNetworkDataSourceAdapter` is an adapter which provides the required methods to handle the network requests.
+# Coroutine Network Adapters
 
-## Coroutine Page Fetcher
+The Coroutine Network Adapters are structures, based on [Kotlin Coroutine](https://kotlinlang.org/docs/reference/coroutines-overview.html) engine, that are used to handle the network request.
```

Comment:
Is `Coroutine` ok? Or should it be `Coroutines`?

Reply:
```suggestion
The Coroutine Network Adapters are structures, based on [Kotlin Coroutines](https://kotlinlang.org/docs/reference/coroutines-overview.html) engine, that are used to handle the network request.
```

Reply:
```suggestion
The Coroutine Network Adapters are structures, based on [Kotlin Coroutines](https://kotlinlang.org/docs/reference/coroutines-overview.html) engine, that are used to handle the network request.
```